falcon/request.py

```diff
@@ -896,7 +896,13 @@ def cookies(self):
             # NOTE(tbug): We might want to look into parsing
             # cookies ourselves. The SimpleCookie is doing a
             # lot if stuff only required to SEND cookies.
-            parser = SimpleCookie(self.get_header('Cookie'))
+            cookie_header = self.get_header('Cookie', default='')
+            parser = SimpleCookie()
+            for cookie_part in cookie_header.split('; '):
+                try:
+                    parser.load(cookie_part)
```

Comment:
fix later: Calling `load()` multiple times will result in a performance degradation. I'll create an issue to address this in 1.4.